public/images/aoe2cm2.svg

```diff
@@ -1,300 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
No need to optimize this one, that's just the source file for the png

Reply:
I removed the file from the commit and added the rest in 74838ce2d6dc8de17a2a47d88109a7ee9adb1205